cmd/frontend/graphqlbackend/user_usage_stats.go

```diff
@@ -153,6 +158,13 @@ func (r *schemaResolver) LogEvents(ctx context.Context, args *EventBatch) (*Empt
 			continue
 		}
 
+		// On Sourcegraph.com only, log a HubSpot event indicating when the user installed a Cody client.
+		if args.Event == "CodyInstalled" && userID != 0 && userPrimaryEmail != "" && envvar.SourcegraphDotComMode() {
```

Comment:
nit: IMHO reads better in the same order as described in a doc above:
```suggestion
		if envvar.SourcegraphDotComMode() && args.Event == "CodyInstalled" && userID != 0 && userPrimaryEmail != "" {
```

Reply:
Ack, thanks